components/plurals/src/rules/runtime/ast.rs

```diff
@@ -219,7 +219,7 @@ fn get_modulus(input: u32) -> Option<reference::ast::Value> {
 
 impl From<&reference::ast::Value> for u32 {
     fn from(v: &reference::ast::Value) -> Self {
-        #[allow(clippy::expect_used)] // TODO(#1668) Clippy exceptions need docs or fixing.
+        #[allow(clippy::expect_used)] // documented
```

Comment:
Question: Where is the documentation?

Reply:
in the panic message right below.

Reply:
It's not documented in the _function signature_, though.

The comment should explain why this panic suppression is consistent with the ICU4X style guide of "core library code doesn't panic". Pointing to a message that says "Failed to convert u64 into u32" doesn't answer that.

Reply:
fixed